docs/README.md

```diff
@@ -1,105 +1,411 @@
-# Geolocation GraphQL Interface
+# Geolocation GraphQL interface
 
-This app exports a GraphQL schema for geolocation results for Place Components.
+>⚠️ **USE AT YOUR OWN RISK**
+> 
+> *This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.*
 
-## Usage
+The Geolocation GraphQL interface provides autosuggestions for address inputs and autocompletion for address forms.
 
-### Installation
+>ℹ️ The Geolocation GraphQL interface allows you to choose which geolocation provider (e.g., [Google Maps](https://www.google.com.br/maps/), [Algolia Places](https://community.algolia.com/places/), [VTEX Places](https://github.com/vtex-apps/places-graphql), etc.) suits better for your store.
 
-To use it in your app, run the `vtex add vtex.geolocation-graphql-interface` command.
+It's possible to use the Geolocation GraphQL interface to connect with any front-end component, adapting the queries according to the UI needs.
 
-You may then use it in your front end component queries, for example, by writing the file `suggestAddresses.graphql`:
+Check the following sections for more information on how to use the data fetched from the Geolocation GraphQL interface in your React components.
 
-```graphql
-query SuggestAddresses($searchTerm: String!) {
-  suggestAddresses(searchTerm: $searchTerm)
-    @context(provider: "vtex.geolocation-graphql-interface") {
-    description
-    mainText
-    mainTextMatchInterval {
-      offset
-      length
-    }
-    secondaryText
-    externalId
-  }
-}
-```
+## Usage examples
```

Comment:
This and other info (including the sections below Usage examples) about "how to use the app" should be under the `Modus Operandi` section.